lib/webauthn/authenticator_data.rb

```diff
@@ -62,35 +72,31 @@ def credential
       end
     end
 
-    def sign_count
-      @sign_count ||= data_at(SIGN_COUNT_POSITION, SIGN_COUNT_LENGTH).unpack('L>')[0]
-    end
-
     def attested_credential_data
       @attested_credential_data ||=
-        AttestedCredentialData.new(data_at(attested_credential_data_position))
+        AttestedCredentialData.deserialize(trailing_bytes)
+    rescue AttestedCredentialDataError
```

Comment:
Should this be `WebAuthn::AttestedCredentialDataFormatError` to be consistent with `AuthenticatorDataFormatError`?

Reply:
Yes!